docs/integration_resources/software/resolve.md

```diff
@@ -1,5 +1,63 @@
-# Resolve
+# Davinci Resolve
 
-Shift integration in Resolve is on the works! 
+## Running Shift in Davinci Resolve
 
-If you want to go ahead and give a try to our current development for Resolve, please let us know through our [contact page](https://inbibo.co.uk/contact?_gl=1*m7h22j*_up*MQ..*_ga*MTU4MzcxNTg4OS4xNzI0MzE2MTQ2*_ga_DY05R1LZDB*MTcyNDMxNjE0Ni4xLjEuMTcyNDMxNjE1Mi4wLjAuMTg0MTg0MzEwNw..).
\ No newline at end of file
+Shift can be integrated in Davinci Resolve and used to automatize editorial process using all the catalog provided with Shift, but specially the Shift_Resolve catalog.
+
+<figure>
+      <img src="images/resolve_shift_ui.png" alt="Resolve Shift UI">
+      <figcaption><b>Figure 1</b>: Shift UI in Davinci Resolve.</figcaption>
+</figure>
+
+To run Shift inside Davinci Resolve is required to set up the configuration required by Blackmagic to have the ability of execution python code inside Resolve. 
+
+## Python Interpreter Setup
+
+Blackmagic usually provides a `Readme.txt` file where these requirements are detailed for the version in use in the installation folder `C:\ProgramData\Blackmagic Design\DaVinci Resolve\Support\Developer\Scripting\Readme.txt`
+
+Apart from the official requirements, the `PYTHONHOME` environment is required to be set. It must point to the python interpreter to be used in Resolve.
+
+To use the Python interpreter provided with Shift, set up the variable in this way:
+
+**SHIFT_PROCESS_NUKE** : <path_to_your_shift_folder>/shift/thirdparty/python

```

Comment:
```suggestion
**SHIFT_PROCESS_RESOLVE** : <path_to_your_shift_folder>/shift/thirdparty/python
```

Reply:
ups, thx for flagging it, but is PYTHONHOME, we can not execute Resolve with the WorkflowProcess. I will fix, thx!

Reply:
done